pkg/kratos/service.go

```diff
@@ -129,7 +143,7 @@ func (s *Service) CreateBrowserRecoveryFlow(ctx context.Context, returnTo string
 	return flow, resp.Cookies(), nil
 }
 
-func (s *Service) CreateBrowserSettingsFlow(ctx context.Context, returnTo string, cookies []*http.Cookie) (*kClient.SettingsFlow, []*http.Cookie, error) {
+func (s *Service) CreateBrowserSettingsFlow(ctx context.Context, returnTo string, cookies []*http.Cookie) (*kClient.SettingsFlow, *http.Response, error) {
```

Comment:
I don't like having to change the service function's signature. If we end up changing it, (imho) we will need to do it for all the service functions.

Reply:
I agree with Nikos, the response from.kratos.should be dealt with at the service layer level